doc/specs/collages.md

```diff
@@ -0,0 +1,140 @@
+# Collages
+
+## Table of contects
+
+1. Motivation
+1. Business rules
+1. Use cases
+
+## Motivation
+
+The collage system is an important feature.
+It provides an orthogonal way to either artist page or tagging system to organise Album, based (mostly) on Facts.
+
+## Business rules
+
+* Collage contains albums
```

Comment:
@salas-wcd A collage should be able to store just about anything we throw at it. Artists, Torrents, Torrent Groups, Requests, Labels, etc. Should be as generic as possible.